account-integrations/safe/test/forge/SafeZkEmailRecoveryPlugin.t.sol

```diff
@@ -148,23 +221,32 @@ contract SafeZkEmailRecoveryPluginTest is TestHelper {
 
         // Act
         vm.startPrank(safeAddress);
-        safeZkEmailRecoveryPlugin.addRecoveryHash(
+        vm.expectEmit(true, true, false, false);
```

Comment:
See the [foundry cheatcode docs](https://book.getfoundry.sh/forge/cheatcodes?highlight=expectEmit#cheatcodes) for more info on how `expectEmit` works